docs/source/interfaces/ISIS_Reflectometry.rst

```diff
@@ -363,7 +362,7 @@ Measure based search transfer uses the log-values within nexus files from the ex
 
 - Any runs with the measurement_id log, will be
   placed into the same group.
-- Any runs with the ``same measurement_id`` and the same ``measurement_subid`` logs, will be merged into a single row, with all the runs listed in the **Run(s)** column in the format, ``123+124+125``. 
+- Any runs with the ``same measurement_id`` and the same ``measurement_subid`` logs, will be merged into a single row, with all the runs listed in the **Run(s)** column in the format, ``123+124+125``.
```

Comment:
Again, nothing you've modified, but could we replace "``same measurement_id``" with "same ``measurement_id``"?